src/NuGet/Microsoft.CodeAnalysis.Common.nuspec

```diff
@@ -34,6 +34,7 @@ Supported Platforms:
             <dependency id="System.Dynamic.Runtime" version="4.0.11-rc2-23910" />
```

Comment:
I think between actual shipped releases that might work. But pre-release builds aren't necessarily compatible with other pre-release builds. For example all the new TFM stuff came in between these 2 pre-release builds. That is not backwards compatible with older pre-release builds.